Makefile

```diff
@@ -62,9 +62,9 @@ gen-k8s: $(apis_sources)
 gen-k8s-check: $(apis_sources)
 	./hack/verify-codegen.sh
 
-container-build: container-build-operator container-build-registry
+container-build: csv-generator container-build-operator container-build-registry
 
-container-build-operator:
+container-build-operator: csv-generator
```

Comment:
remove `csv-generator` so CI will pass.

Reply:
The csv-generator make target is required for the operator container. It will make CI pass, but csv-generator will not be configured correctly to run within the operator's container.